crates/rooch-rpc-client/src/client_config.rs

```diff
@@ -21,6 +21,8 @@ pub const ROOCH_TEST_NET_URL: &str = "https://test-seed.rooch.network:443/";
 #[derive(Serialize, Deserialize, Debug)]
 pub struct ClientConfig {
     pub keystore_path: PathBuf,
+    pub password_hash: Option<String>,
+    pub is_password_empty: bool,
```

Comment:
`password_hash` and `is_password_empty` should be part of `ClientConfig` or `KeyStore`?

Reply:
Okay. I overlooked it. It looks like it can be moved to `BaseKeyStore`.